drivers/afe/ad4110/ad4110.c

```diff
@@ -551,15 +551,11 @@ int32_t ad4110_spi_int_data_reg_read(struct ad4110_dev *dev,
 
 	data_size = ad4110_get_data_size(dev, A4110_ADC, AD4110_REG_DATA);
 
-	buf[0] = (A4110_ADC << 7) | AD4110_CMD_READ_COM_REG(
-			 AD4110_REG_DATA); // cmd byte
+	buf[0] = (A4110_ADC << 7) |
+		 AD4110_CMD_READ_COM_REG(AD4110_REG_DATA); // cmd byte
 	buf[0] |= (dev->addr << 4) & AD4110_DEV_ADDR_MASK;
 
-	buf[1] = 0xAA; // dummy data byte
-	buf[2] = 0xAA; // dummy data byte
-	buf[3] = 0xAA; // dummy data byte
-	buf[4] = 0xAA; // dummy data byte
-	buf[5] = 0x00; // dummy data byte
+	memset(buf + 1, 0xAA, 5); // dummy data bytes
```

Comment:
This can be fixed in the first commit. Because there is where you created the function